http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/HeadersResource.java

```diff
@@ -28,4 +30,11 @@ public Uni<Response> headersOverride() {
         return Uni.createFrom().item(response);
     }
 
+    @GET
+    @Path("/no-accept")
+    @Produces(MediaType.TEXT_PLAIN)
+    public Uni<String> okHeaders() {
```

Comment:
Thinks that here you should return `Uni<CustomHeaderResponse>` ti intercept and modify the response